README.md

```diff
@@ -340,6 +341,28 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+### Implicit Getters ###
+
+When overriding only the getter of a property, omit the use of `get`:
```

Comment:
It's picky but I feed that without `get` it's harder to read when you have multiple lines